app/src/main/java/org/jellyfin/androidtv/ui/AlphaPicker.kt

```diff
@@ -0,0 +1,42 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.Button
+import android.widget.LinearLayout
+import org.jellyfin.androidtv.R
+
+class AlphaPicker(
+	context: Context,
+	attrs: AttributeSet?
```

Comment:
```suggestion
	attrs: AttributeSet? = null
```

Automatically creates a secondary constructor with a null value